### DISCUSSION TITLE
jQuery Migrate

### DISCUSSION THREAD

**Discussion by twachs31** [jquery/jquery-migrate#490] [Q&A]
I am attempting to upgrade my jQuery 3.3.1 to your most recent version (3.7.0).  I am using migrate to make the transition smooth.  I downloaded jquery-3.7.0.js, jquery-3.7.0.min. js, jquery-migrate-3.4.0.js and jquery-migrate-3.4.0.min.js, All four had this code in line two:
     "undefined" == typeof jQuery.migrateMute && (jQuery.migrateMute = !0),
Looking at the documentation, I see that it has something to do with suppressing errors and/or warnings.  However, for me, it caused an error, "Uncaught ReferenceError: Jquery is not defined".  Can you tell me what this line truly does and what I can do to fix it?  I am assuming that this line would not go to production.   I amusing Chrome, Edge, and sometimes IE 11+. So, maybe I can live without it.  Let me know.  Thank you.

**Comment by mgol** [comment]
`migrateMute` is not referenced in jQuery, only in jQuery Migrate. So if you see it in all 4 files, they must be messed up.

You need to load jQuery before jQuery Migrate.

**Comment by twachs31** [comment]
Morning.  Thank you for responding.  I was able to get passed the last error.

Now, I am getting this error

***@***.***

It says, “Uncaught ReferenceError: jQuery is not defined”

**Reply by mgol** [reply]
That’s the same error.

**Comment by twachs31** [comment]
Yes.  It is the same error message, but it is occurring in a different place in the code.   I do not see where jQuery is defined.  It is also not defined in “function ( jQuery, window )”

**Reply by mgol** [reply]
This just means you still don't include jQuery before Migrate correctly. It's hard to guess anything more just from the error message, you need to provide more information, preferrably an isolated standalone test case.
